notebooks/pluvial/Nested_Hyetograph.ipynb

```diff
@@ -0,0 +1,1158 @@
+{
```

Comment:
Here are my bulleted comments for this notebook:

- This notebook is nearly identical to PrecipTable.ipynb. Instead of having two notebooks that share 90% of the same code, I would think it would be better to just have a single notebook that can do both things. The only differences I really see are the durations, not including the upper and lower bounds when pulling the precipitation frequency data, and the additional calculation for the nested hydrograph at the end. This additional calculation could just be its own cell. 
- Under "Determine Projection" remove vector_crs_prj = gdf.crs. This ignores the entire point of the cell it is within, which is to identify the native projection or assign one if it does not exist. 
- I updated hydromet.py so that the crs is converted to a projected coordinate system before performing the intersection in order to calculate the volume and region code (this gives us a better estimate of the percent area).
- May want to consider specifying the NRCS nested hyetographic function events instead of using a slice of the index since this seems like it may not always be consistent for all NOAA Atlas 14 volumes and regions.

Reply:
Nested_hyetograph notebook to be merged with PrecipTable.  Note that remove vector_crs_prj = gdf.crs has been deleted. The hydromet.py projection issue with NOAA Atlas 14 has been resolved. Change c_df.index.astype(str).tolist()[1:10] to a list of values defined by the user, so that the code checks for each value (i.e., it occurs from the NOAA Atlas 14 data). Plots need axes and overall labels.

Reply:
To be done

Reply:
- Nested_hyetograph notebook is now merged with PrecipTable. 
- c_df now based on user input.
- plots have labels and axes.